applications/asset_tracker_v2/configuration/memfault/memfault_metrics_heartbeat_config.def

```diff
@@ -0,0 +1,7 @@
+/* Application-specific metrics can be defined here.
+ * Please refer to https://docs.memfault.com/docs/embedded/metrics-api for more details.
```

Comment:
I like that it contains the links to the Memfault docs. But I don't have any strong opinion on this. I can remove it if you think its unnecessary.

Reply:
Yeah, it has some value to have the link. It was mostly the ` can be defined here` part I found a bit misleading as there's already some metrics here. Maybe that can be cut, so that it reads `Application-specific metrics`, but it's not a bug deal. Don't push and re-run CI just for this change